view/index.html

```diff
@@ -40,7 +40,7 @@ <h3 class="crunch-top"><span class="pull-left glyphicon glyphicon-exclamation-si
   {{#deleted}}
       <div class="col-md-12 clearfix" data-test="alert">
           <div class="alert alert-info">
-              <button aria-hidden="true" data-dismiss="alert" class="close" type="button">×</button>
+              <button data-dismiss="alert" class="close" type="button">×</button>
```

Comment:
```suggestion
              <button data-dismiss="alert" class="close" type="button" aria-label="Close">×</button>
```

OH THE SUGGESTION BUTTON ONLY APPEARS IN THE "FILES CHANGED" VIEW, that's why it wasn't there (I was looking at the commits view)